### DISCUSSION TITLE
NLP - PAT Rubric

### DISCUSSION THREAD

**Discussion by jlooper** [microsoft/ML-For-Beginners#65] [PAT - Progress Assessment Tools]
| Lesson | Learning Targets | Strengths | Self-Reflection and Feedback |
|---|---|---|---|
| 1 | Learn about the history underpinning NLP |  Build a very basic conversational bot |  |
| 2 | Understand common tasks suitable for NLP and the problems it can solve | Build your NLP vocabulary, enhance your bot |  |
| 3 | Build your understanding of sentiment analysis using NLP | Analyze texts from Pride and Prejudice for sentiment polarity |  |
| 4 | Working with a large corpus | Understand, prepare, and clean hotel review data |  |
| 5 | Learn about NLTK for NLP processing | Complete the data processing and perform sentiment analysis |  |

**Comment by Anonymoushacker5** [comment]
How does it look like

**Reply by jlooper** [reply]
you can add your own progress here
